### PR TITLE
Add check for pushurl arg

### PR DESCRIPTION
**Pull request by fermadeiral**
This PR intends to fix issue #168.
I thought on two ways to fix the mentioned issue:
1) To try to repair the push url in the push step (last step of the pipeline)
2) To check the correctness of the push url when other variables are also checked (before the pipeline starts)

I took the second way, since the first way might not work depending on how wrong the push url is, and the user of repairnator would know there was a problem at pushing only in the end of the execution.
WDYT?

PS: the build is passing in https://github.com/fermadeiral/repairnator/pull/8

### CODE REVIEW

File: repairnator/repairnator-core/src/test/java/fr/inria/spirals/repairnator/TestUtils.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    @Test
+    public void testMatchesGithubRepoUrlWithWrongRepoUrl() {
+        String repoUrl = "https://github.com/Spirals-Team/repairnator/";
+
+        assertThat(Utils.matchesGithubRepoUrl(repoUrl), is(false));
+    }
+
+    @Test
+    public void testMatchesGithubRepoUrlWithWrongRepoUrl2() {
```
Comment (surli) [inline repairnator/repairnator-core/src/test/java/fr/inria/spirals/repairnator/TestUtils.java]: Could you add a test with project containing numbers?

Comment (fermadeiral) [reply repairnator/repairnator-core/src/test/java/fr/inria/spirals/repairnator/TestUtils.java]: Sure. It's done, and it works for numbers.

However, I just realized that repo name can contain dot, and also I checked the rule for user name ("Username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen and is already taken" from GitHub). I will do some changes for these things. I tell you when it's done.

**Comment by fermadeiral** [comment]
@surli this PR is ready again.

**Comment by surli** [comment]
Great thanks!

**Pull request merged**